frontend/src/bundles/home/pages/HomePage.vue

```diff
@@ -0,0 +1,264 @@
+<template>
+  <div>
+    <HeaderLandingNav
+      :items="itemsMenu"
+      :is-authorized="isAuthorized"
+    />
+    <v-main>
+      <v-parallax
+        height="900"
+        src="@/assets/image/home_bg_header_no_harts.png"
+      >
+        <v-container>
+          <v-row
+            align="center"
+            justify="center"
+          >
+            <v-col
+              class="text-center"
+              cols="10"
+              md="4"
+              sm="6"
+            >
+              <h1 class="main--text h1 black--text font-weight-bold pb-md-4 pb-2">
+                It is never too late
+                to fall in love
+              </h1>
+              <p class="main-sub--text subheading black--text pb-md-2 pb-2">
+                When you find that one that's right for you,
+                you feel like they were put there for you,
+                you never want to be apart.
+              </p>
+              <v-btn
+                class="white--text text-capitalize font-weight-bold"
+                color="button-main"
+                large
+                rounded
+                depressed
+                to="/auth/register"
```

Comment:
Instead of a path better to provide a route name for navigation.
```suggestion
                to="{ name: 'auth-registration-set_up' }"
```

Reply:
fixed